prow/plugins/override/override.go

```diff
@@ -336,6 +346,30 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent,
 		return oc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, user, resp))
 	}
 
+	checkruns, err := oc.ListCheckRuns(org, repo, sha)
```

Comment:
This has to be gated by a check if github apps auth is used, as otherwise it is not allowed to write to checkruns, so listing them is pointless

Reply:
This is still not addressed. We should not waste api tokens on this when we can not create check runs.

Reply:
Not sure how to do that.

Reply:
The client has this information: https://github.com/kubernetes/test-infra/blob/b74506333229b4cecfddccded988e80f5742e4b3/prow/github/client.go#L314

Looks like we need to add a getter for that though, I thought we already expose it but seems I misremember

Reply:
Fixed.